invoices/sql_store.go

```diff
@@ -1116,6 +1116,9 @@ func (s *sqlInvoiceUpdater) AddAmpHtlcPreimage(setID [32]byte,
 			SetID:     setID[:],
 			HtlcID:    int64(circuitKey.HtlcID),
 			Preimage:  preimage[:],
+			ChanID: strconv.FormatUint(
```

Comment:
Does this mean we previously may have updated the wrong htlc?

Reply:
Yes, with sqlite we updated multiple HTLCs at once with the same preimage. With postgres, we never updated any preimages at all.

Reply:
I think this only affects AMP HTLCs - could you add a release note on this fix?

Reply:
Yes, only AMP HTLCs with multiple HTLCs sharing a set ID. Will update release notes.

Reply:
Updated release notes.